libs/designsystem/src/lib/scss/_global-styles.scss

```
@@ -129,17 +129,17 @@ ion-modal.kirby-overlay {
           --min-height: #{$modal-default-height};
           align-items: flex-start;
 
-          &.small {
+          &.kirby-modal--small {
             --min-height: #{map-get($modal-heights, 's')};
           }
-          &.medium {
+          &.kirby-modal--medium {
             --min-height: #{map-get($modal-heights, 'm')};
           }
-          &.large {
+          &.kirby-modal--large {
```

Please align with existing naming:
```suggestion
          &.kirby-modal-large {
```